api/controllers/media.js

```diff
@@ -1,6 +1,8 @@
 const uuidv1 = require('uuid/v1');
 const azure = require('azure-storage');
-const blobService = azure.createBlobService();
+const blobService = azure.createBlobService(
```

Comment:
You Must not commit this. It should be a local change in your own development environment

Reply:
@aljvilla please undo this update